luke/index.js

```diff
@@ -0,0 +1,23 @@
+// Q1.In one night, a movie theater sells tickets for 6450 dollars.Each ticket costs 15 dollars.How many tickets were sold ?
+let tiketsSold = 6450 / 15;
+console.log(tiketsSold);
+
+// Q2.Sylvia's income is 500 dollars per week. How much does Sylvia makes every year?
+let yearIncome = 500 * 12;
```

Comment:
Her salary is 500 dollars per week and not month, therefore you have to multiply 500 with 52(weeks per year)

Reply:
Hey yeup just made that change will push it soon :)